### DISCUSSION TITLE
Custom link in notifications

### DISCUSSION THREAD

**Discussion by u12206050** [directus/directus#15777] [Feature Requests]
Currently notifications work great if the intention is to direct the user's attention to a specific item in a collection.
However when you want to alert the user to something else then the current link functionality is quite useless.

I propose updating the Notification table with a link property that can be set to any link (even external) and have the frontend navigate to that page instead if set instead of the collection/item.

Two ways to achieve this
1. either replace the `collection` and `item` properties with one `link` field (that does support relative links eg. `/admin/content/places/123`)

2. Add an additional link property

What could be cool is when you want to link to a say a custom module, and have the item id as query parameter.
eg.```
collection: 'registrations',
item: '5123',
link: '/admin/custom-registrations-module'
```

Would direct you to `/admin/custom-registrations-module?item=5123&collection=registrations`

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
